### PR TITLE
Improved code base

### PR DESCRIPTION
**Pull request by GlaciErrDev**
- Bumped python version to 3.8
  - After each stream.write() added missing coroutine [stream.drain()](https://docs.python.org/3.8/library/asyncio-stream.html#asyncio.StreamWriter.drain)
  - After each stream.close() added missing coroutine [stream.wait_closed()](https://docs.python.org/3.8/library/asyncio-stream.html#asyncio.StreamWriter.wait_closed)
  - Added timeouts to each stream reader
    by default it's 10s

### CODE REVIEW

**Comment by joker-at-work** [comment]
I'd preferred it, if you would have put each of the bullet points in your one commit into their own commit. They seem to be independent entities after all. If we see that the timeout is bad, we have to go through the code again instead of just reverting a single commit.
Having different commits also gives you the opportunity to explain each change a little better, e.g. what happens if we don't drain/close and what are we trying to fix with a 10s timeout. It's always best to get down as many thoughts as possible while they're fresh. I can promise you, the same topic will come up later (in a year?) and you will not remember the reasoning. Happens all the time.

EDIT: Also, please rebase now that the first commit is in master.

File: vspc/server.py [hunk 1]
```diff
@@ -202,20 +207,26 @@ async def handle_telnet(self, reader, writer):
...
                 await self.save_to_log(uuid, data)
-                data = await telnet.read_some()
+                async with timeout(10):
+                    data = await telnet.read_some()
         finally:
             self.sock_to_uuid.pop(socket, None)
-        LOG.info("%s disconnected", peer)
-        writer.close()
+            LOG.info("%s disconnected", peer)
+            await writer.drain()
```
Comment (joker-at-work) [inline vspc/server.py:227]: What happens if the other side of our socket is gone? Where does it drain to?

Comment (GlaciErrDev) [reply vspc/server.py:227]: We'll have `ConnectionError`... will make the check here.

**Comment by GlaciErrDev** [comment]
> I'd preferred it, if you would have put each of the bullet points in your one commit into their own commit. They seem to be independent entities after all. If we see that the timeout is bad, we have to go through the code again instead of just reverting a single commit.
> Having different commits also gives you the opportunity to explain each change a little better, e.g. what happens if we don't drain/close and what are we trying to fix with a 10s timeout. It's always best to get down as many thoughts as possible while they're fresh. I can promise you, the same topic will come up later (in a year?) and you will not remember the reasoning. Happens all the time.
> 
> EDIT: Also, please rebase now that the first commit is in master.

Agree! I will split it.

**Pull request closed without merging**